test/indent.txt

```diff
@@ -101,6 +163,13 @@ to foo ; comment
 + ask turtles [ fd 1 ]
 end
 
+Comments3
```

Comment:
I can rewrite this case. This case is designed to test something that the prior comment tests do not test, which is:

* multi-line comment alignment
* bottom comment line starts out of alignment
* top comment line starts out of alignment

Because we now make all indentation changes in a single pass (rather than making changes, re-evaluating, making more changes) this case is particularly fragile and needs a test. Having said that, I agree that this case doesn't actually look like code someone would write.